web/src/drafts.js

```diff
@@ -638,6 +660,12 @@ export function launch() {
             update_bulk_delete_ui();
         });
 
+        $("#drafts_table .overlay_message_controls .copy-overlay-message").on("click", function () {
+            const $draft_row = $(this).closest(".overlay-message-row");
+            const $draft_id = $draft_row.data("draft-id");
```

Comment:
```suggestion
            const draft_id = $draft_row.data("draft-id");
```

Since this is not a jQuery element, there is no need for the `$` prefix.

Reply:
Changed, Thanks for noticing